.php_cs.dist

```diff
@@ -1,10 +0,0 @@
-<?php
-
-$finder = PhpCsFixer\Finder::create()
-    ->in('src')
-    ->name('*.php')
-    ->exclude('src/Migrations/*')
-;
-
-include __DIR__.'/.php_cs_rules.dist';
```

Comment:
Any specific reason why this is split up? To change only the finder whilst still keeping the rules most likely?

Reply:
Yes indeed, depending on if it is a bundle or a project, you would need different settings